website/addons/dropbox/templates/dropbox_node_settings.mako

```diff
@@ -1,136 +1 @@
-                    <div data-bind="visible: currentDisplay() === SHARE && emails().length">
-                        <div class="help-block">
-                            <p>To share this folder with other Dropbox users on this project, copy
-                            the email addresses of the contributors (listed below) into the
-                            "Share Folder" dialog on Dropbox.</p>
-                        </div>
-
-                        <label for="contrib-emails">Copy these:</label>
-                        <div class="input-group">
-                            <textarea name="contrib-emails"
-                                    class="form-control" rows="3" id="contribEmails"
-                             data-bind="value: emailList,
-                                        attr.autofocus: currentDisplay() === SHARE">
-                            </textarea>
-                            <span data-clipboard-target="contribEmails"
-                                class="input-group-addon pointer"
-                                id="copyBtn">
-                                <i class="icon-paste"></i>
-                            </span>
-                        </div>
-
-                        <div class="input-group pull-right">
-                            <a target="_blank" data-bind="attr.href: urls().share"
-                                class="btn btn-link"><i class="icon-share-alt"></i> Continue to Dropbox...</a>
-                        </div>
-                    </div>
-
-                    <!-- Queued selection -->
-                    <div class="dropbox-confirm-selection"
-                        data-bind="visible: currentDisplay() == PICKER && selected()">
-                        <form data-bind="submit: submitSettings">
-                            <div class="pull-right">
-                                <button class="btn btn-default"
-                                        data-bind="click: cancelSelection">
-                                    Cancel
-                                </button>
-                                <input type="submit"
-                                       class="btn btn-primary"
-                                       value="Submit" />
-                            </div>
-                            <h4 data-bind="if: selected" class="dropbox-confirm-dlg">
-                                Connect &ldquo;{{ selectedFolderName }}&rdquo;?
-                            </h4>
-                        </form>
-                    </div><!-- end .dropbox-confirm-selection -->
-
-                </div>
-            </div><!-- end col -->
-        </div><!-- end row -->
-    </div><!-- end .dropbox-settings -->
-
-    <!-- Flashed Messages -->
-    <div class="help-block">
-        <p data-bind="html: message, attr.class: messageClass"></p>
-    </div>
-</div><!-- end #dropboxScope -->
+${context.lookup.get_template('project/addon/folder_picker.mako').render(**context.kwargs)}
```

Comment:
This can be made nicer as soon as https://github.com/CenterForOpenScience/osf.io/pull/2104 is merged